os/net/netdev/netdev_ioctl.c

```diff
@@ -1114,7 +1146,7 @@ int netdev_ioctl(int sockfd, int cmd, unsigned long arg)
 	 * non-NULL.
 	 */
 
-	if (!_SIOCVALID(cmd)) {
+	if (((cmd != FIONREAD) && (cmd != FIONBIO)) && !_SIOCVALID(cmd)) {
```

Comment:
To use && two times is correct..?
In this logic, cmd which is either FIONREAD or FIONBIO is false.

Reply:
It's correct. If both FIONREAD and FIONBIO are false and _SIOCVALID(cmd) is also false, it shows an error.

Reply:
Okay, I understand.
It is the same as below.
if (!(cmd == FIOREAD || cmd == FIONBIO || _SIOCVALID(cmd)) {
...

Reply:
That's more clear. I will change the code.